src/vmm_mad/remotes/lib/lxd/mapper/mapper.rb

```diff
@@ -552,4 +557,13 @@ def update_partable(dev)
         Command.execute(cmd, false)
     end
 
+    def mkfs(device, format)
```

Comment:
Does this works for ceph volatiles, sudo needed here?

Reply:
There are no ceph volatiles, only raw, qcow2 and swap. sudo not really needed, it's just mkfs being on `/sbin/`. Command pointer is `:mkfs       => '/sbin/mkfs'`